### PR TITLE
Add Mini Board and Podium to Tournaments

### PR DESCRIPTION
**Pull request by freefal**
Addresses features in tournament issue (https://github.com/veloce/lichobile/issues/283)

### CODE REVIEW

File: project/src/js/ui/tournament/detail/tournamentDetailView.js [hunk 1]
```diff
@@ -200,3 +223,59 @@ function renderFooter(ctrl) {
...
+function renderPlace(data) {
+  const rank = data.rank;
+  return (
+    <div className={'place'+rank}>
+      <div className="trophy"> </div>
+      <div className="username" config={helper.ontouch(() => m.route('/@/' + data.name))}>
+        {data.name}
+      </div>
+      <div className="rating"> {data.rating} {helper.progress(data.ratingDiff)} </div>
+      <table className="stats">
```
Comment (veloce) [inline project/src/js/ui/tournament/detail/tournamentDetailView.js]: I'd rather not have a table here. It's really easy to do in flex, I can do it if you want.

**Comment by veloce** [comment]
👍 
really cool

There are just some style adjustments to do regarding the podium. I can do that if you want, it's easy for me. So I can merge this PR right away.

**Comment by freefal** [comment]
Great. Just pulled your changes. Definitely an improvement.
I used the table in that last section because I was copying thibault front-end html. Do you still want me to switch it to flex and send you another PR? Happy to do it if so.

**Comment by veloce** [comment]
No it's ok, let's keep it that way.

**Comment by freefal** [comment]
Great. By the way, I really like the flexbox layout. Still lets me think in terms of rows and columns and very flexible (no pun intended).

**Pull request merged**